livekit-agents/livekit/agents/transcription/transcription.py

```diff
@@ -0,0 +1,87 @@
+from typing import Callable
+
+from livekit import rtc
+
+from .. import tokenize
+from .stt_forwarder import STTSegmentsForwarder
+from .tts_forwarder import TTSOptions, TTSSegmentsForwarder
+
+
+class TranscriptionManager:
+    def __init__(self, room: rtc.Room):
+        self._room = room
+
+    def forward_stt_transcription(
```

Comment:
`forward*` sounds like it's actually performing the forwarding, which it does not.

```suggestion
    def create_stt_forwarder(
```